Makefile

```diff
@@ -196,7 +196,6 @@ update-otel:
 	$(MAKE) update-dep MODULE=go.opentelemetry.io/contrib/detectors/gcp VERSION=$(STABLE_CONTRIB_OTEL_VERSION)
 	$(MAKE) update-dep MODULE=go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp VERSION=$(UNSTABLE_CONTRIB_OTEL_VERSION)
 	$(MAKE) update-dep MODULE=go.opentelemetry.io/collector VERSION=$(COLLECTOR_VERSION)
-	$(MAKE) update-dep MODULE=go.opentelemetry.io/collector/pdata VERSION=$(COLLECTOR_VERSION)
```

Comment:
why is this removed?

Reply:
`pdata` made it to `v1.0.0` on the `v0.67.0` [release](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.67.0)

Reply:
Got it.  Can you split it into stable collector version and unstable collector version instead of removing?